composer.json

```diff
@@ -36,7 +36,9 @@
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
         "symfony/browser-kit": "^3.2",
         "symfony/finder": "^3.2",
-        "symfony/framework-bundle": "^3.2"
+        "symfony/framework-bundle": "^3.2",
+        "alcaeus/mongo-php-adapter": "1.0.*",
```

Comment:
It should be an additional dependency(probably together with `doctrine` but it is another topic)